data/remnant/remnant 2 side missions.txt

```diff
@@ -0,0 +1,146 @@
+			scene "scene/remnant remote spaceport"
+			`	Taely looks around with pride, and then watches your reaction with amusement.`
+			choice
+				`	"I do not know what to make of this."`
+				`	"This is a spaceport?"`
+			`	She nods, and guides you amongst the stream of camel-drawn hovercarts heading for the gate. "This is one of our earliest spaceports, before we learned new techniques better suited to our new home. Our founding ships brought as much as they could, of course, but as it turns out, the earliest phases of expansion look much the same whether one is looking at historic pre-flight eras, or modern settlement efforts."`
+			scene "scene/remnant remote spaceport"
+			label questionloop
+			choice
+				`	"It still seems odd to see camels pulling levitating platforms."`
+					goto camelcart
+				`	"Stone walls seem like an odd thing to build."`
+					goto stonewalls
+				`	"Are those wind turbines?"`
+					goto windturbine
+				`	"All these wood products seems surprising."`
+				`	"I think I have seen enough."`
+					goto mainreturn
+			label woodproducts
+			scene "scene/remnant remote spaceport"
+			`	Taely picks up a board from a nearby pile. "We were fortunate that Caelian had some vegetation that could be used like this. The native vegetation is biologically quite different from the trees of Earth, but they have similar structural characteristics and grow plentifully in many areas." She offers you the board. "The decision was made early on that anything that could be made from locally sourced materials, should be sourced locally. We wanted to conserve our limited supplies of metals and other rare resources for things that required it." She taps on a barrel. "These are a prime example - a few metal straps enable the wood to hold a significant amount of liquid. It is much the same as how a comparatively small amount of bone in our bodies can provide the necessary structure for a large volume of soft tissues and organs to function."`
+				goto questionloop
+			label camelcart
+			scene "scene/remnant remote spaceport"
+			`	"Is that so? The engineering advantages are quite simple: With the mechanics of the cart focused entirely on supporting the load and the camel providing horizontal motion, the cart can support much larger masses than it could if it had to split the forces." She looks wistful. "In truth, I am somewhat envious of the industrial capacity the ancestral worlds contain - We have accomplished much, but we have had to make choices, and one of those was the decision as to where to spend our industrial capacity. We chose, obviously, to focus on spacecraft and heavy industrial applications. We have no cars, no personal vehicles at all, really. Even these hovercarts are basically the repulsor systems from our spacecraft that have been repurposed to carry massive loads."`
```

Comment:
```suggestion
			`	"Is that so? The engineering advantages are quite simple: with the mechanics of the cart focused entirely on supporting the load and the camel providing horizontal motion, the cart can support much larger masses than it could if it had to split the forces." She looks wistful. "In truth, I am somewhat envious of the industrial capacity the ancestral worlds contain - we have accomplished much, but we have had to make choices, and one of those was the decision as to where to spend our industrial capacity. We chose, obviously, to focus on spacecraft and heavy industrial applications. We have no cars, no personal vehicles at all, really. Even these hovercarts are basically the repulsor systems from our spacecraft that have been repurposed to carry massive loads."`
```